proto/invocation.proto

```diff
@@ -241,6 +241,9 @@ message InvocationQuery {
 
   // Stat filters (duration_usec, cas_cache_misses, etc.)
   repeated stat_filter.StatFilter filter = 14;
+
+  // The pattern for the targets built. Ex: "//..."
```

Comment:
Since patterns are a little ambiguous do you want to comment that this is an exact match?

Reply:
Sure yeah, done.